### PR TITLE
[HOLD] Remove preservation-ingest-initiated step from accessionWF

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
Part of https://github.com/sul-dlss/argo/issues/1789

## Was the documentation (API, README, DevOpsDocs, wiki, consul, etc.) updated?
no

## Does this change affect how this application integrates with other services?

change was tested on stage

### CODE REVIEW

File: config/workflows/accessionWF.xml [hunk 1]
```diff
@@ -39,19 +39,16 @@
     <label>Initiate Ingest into Preservation</label>
     <prereq>provenance-metadata</prereq>
   </process>
-  <process batch-limit="1" error-limit="5" name="preservation-ingest-initiated" sequence="11" skip-queue="true">
-    <label>Signal that preservation ingest has been initiated</label>
-  </process>
-  <process batch-limit="1" error-limit="5" lifecycle="deposited" name="sdr-ingest-received" sequence="12" skip-queue="true">
+  <process batch-limit="1" error-limit="5" lifecycle="deposited" name="sdr-ingest-received" sequence="11" skip-queue="true">
```
Comment (ndushay) [inline config/workflows/accessionWF.xml:42]: i strongly prefer "preservation" as it is less confusing than "sdr" which means a lot of different things.

Comment (ndushay) [reply config/workflows/accessionWF.xml:42]: also - is this still "skip-queue?"

Comment (jcoyne) [reply config/workflows/accessionWF.xml:42]: Changing the name of the existing step is out of scope.  no, it's not skip queue, we're just removing the one step and renumbering the remaining steps.

Comment (ndushay) [reply config/workflows/accessionWF.xml:42]: this step you are keeping has "skip-queue" and i'm questioning if you still want it to say that.  I suspect you don't.

Comment (jcoyne) [reply config/workflows/accessionWF.xml:42]: Yes, if that’s what it was, we want to keep it. We’re only changing the order attribute

**Pull request merged**